themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -581,6 +581,12 @@ backchannel-logout=Backchannel Logout
 backchannel-logout.tooltip=Does the external IDP support backchannel logout?
 user-info-url=User Info URL
 user-info-url.tooltip=The User Info Url. This is optional.
+client-auth=Client Authentication
+client-auth.tooltip=The client authentication methdod (cfr. https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication).
```

Comment:
There is typo in the tooltip. Could also this tooltip be changed to contain a bit more details. IMO it's not 100% clear, which private key will be used in case of "JWT signed with private key" is chosen. Maybe something like: "The client authentication method (cfr. https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication). In case of JWT signed with private key, the realm private key is used." WDYT?

Reply:
Nicely seen ;) 
I think it's wise to add the information about the private key option like you proposed.
I will update that as soon as possible, should be tonight or tomorrow.